ricecooker/utils/downloader.py

```diff
@@ -360,7 +360,15 @@ def download_assets(
             if node[attr].startswith("data:"):
                 continue
 
-            url = urljoin(base_url, node[attr])
+            if "background-image" in selector:
+                url_match = re.search(r'url\("([^"]+)"\)', node["style"])
```

Comment:
I don't think this will catch all cases of the CSS `url` function, see here for a comprehensive example in JS regex: https://github.com/learningequality/kolibri/blob/develop/packages/kolibri-zip/src/fileUtils.js#L33

Reply:
I'm not going to argue with any regex string. I have adapted the example in js to python and changed this code after I tested it works.